src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -0,0 +1,194 @@
+'use strict';
+
+import React from 'react';
+import _ from 'underscore';
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['_' is defined but never used. (no-unused-vars)](https://app.codacy.com/app/4dn/fourfront/pullRequest?prid=4271272)